mobile/src/core/theme/colors.ts

```diff
@@ -1,13 +1,83 @@
-interface IColors {
-  white: string
-  grey: string
-  black: string
-  blue: string
-}
-
-export const colors: IColors = {
-  white: '#FFFFFF',
-  grey: '#F2F2F2',
-  black: '#262626',
-  blue: '#00579D'
+export enum Colors {
+  transparent = 'transparent',
+  white = '#FFFFFF',
+  black = '#262626',
+  greyLight = '#F2F2F2',
+  grey = '#CDCDCD',
+  greyDark = '#595959',
+  blue = '#00579D',
+  blueDark = '#002341',
+  pink = '#E91E63',
+  red = '#F44336'
+}
+
+export interface ThemeColors {
+  color: Colors;
+  bgColor: Colors;
+  inputBorderColor: Colors;
+}
+
+export enum ColorThemeNames {
+  default = 'default',
+  disabled = 'disabled',
+  light = 'light',
+  dark = 'dark',
+  white = 'white',
+  black = 'black',
+  warning = 'warning',
+  negative = 'negative'
+}
+
+interface Themes {
+  default: ThemeColors,
+  disabled: ThemeColors,
+  light: ThemeColors,
+  dark: ThemeColors,
+  white: ThemeColors,
+  black: ThemeColors,
+  warning: ThemeColors,
+  negative: ThemeColors,
+}
+
+export const themeColors: Themes = {
```

Comment:
Seems kinda weird to group all of these things into a single concept. 

`dark` and `light` make sense as themes, the others seem like mixins or sass variables.